main.js

```diff
@@ -0,0 +1,48 @@
+
+snakewaterMontana = {
+    paleontologist: "Dr. Alan Grant",
+    depth: "10 meters",
+    specimen: "Velocirapto"
+};
+
+var guestOfHonor = snakewaterMontana.paleontologist;
+var cleverGirl = snakewaterMontana.specimen;
+console.log (cleverGirl);
+
+nicaragua = {
+    depth: "200 meters",
+    annualBudget: 1500000,
+    specimens: [
+      "Tyrannosaurus Rex",
+      "Stegosaurous",
+      "Triceratops",
+      "Velociraptor"
+    ]
+  }
+ var specimens = nicaraguanSpecimens 
+ var favoriteSpecimen = nicaraguanSpecimens.velociraptor;
```

Comment:
since velociraptor  is a values inside an array we need another way to access this value we need to know in which index arrays start with index 0 

```
   nicaraguanSpecimens: [
      "Tyrannosaurus Rex", // index 0 
      "Stegosaurous", // index 1
      "Triceratops", // index 2
      "Velociraptor" // index 3 
    ]
```

so the final solution will be something like this ` var favoriteSpecimen = nicaraguanSpecimens[3]`